src/odemis/acq/fastem.py

```diff
@@ -311,6 +342,9 @@ def __init__(self, scanner, multibeam, descanner, detector, stage, ccd, beamshif
                             (model.DataArray): The acquisition data, which depends on the value of the
                                                detector.dataContent VA.
                             (Exception or None): Exception raised during the acquisition or None.
+        :param settings_obs: (SettingsObserver) VAs of all components of which some will be
+                             integrated in the acquired ROA as metadata.
+
```

Comment:
Shouldn't this parameter be between `pre_calibrations` and `future`? That last parameter is a bit special as it's an extra parameter compared to `acquire()`, to track the progress and return the result.